### PR TITLE
added licenses and updated readme

### PR DESCRIPTION
**Pull request by Rag1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,7 @@ Once we have this info, we post an issue comment through an API call.
 
 ### How to run this?
 More info about running the app can be found in [/app](https://github.com/alekhyareddy28/Github-Issue-Insights/tree/master/app)
+
+### References
+We drew inspirations from [Issue label bot](https://github.com/machine-learning-apps/Issue-Label-Bot) and [Gh_app_demo](https://github.com/Mariatta/gh_app_demo).
+The learning model was based on the [word2vec model](https://github.com/alexandres/lexvec)
```
Comment (arjunbalgovind) [inline README.md:21]: "The learning model used the [word2vec model]() for feature extraction from text."

Comment (Rag1) [reply README.md:21]: A tad bit too late :P

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,27 @@
+'''
+MIT License
```
Comment (arjunbalgovind) [inline app/main.py:2]: do we need this on all files? feels overkill

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 {Alekhya Reddy, Arjun Balgovind, Raghavan Sundararaman, Ranjini Chidambaram}
```
Comment (chithams) [inline LICENSE:3]: Pls change to Chithambaram :P

File: app/main.py (hunk 2 above)
Comment (Rag1) [reply app/main.py:2]: I am not sure tbh, if some source code is taken from a repo using a license like MIT it is recommended that we add it to the source file if the binaries are not being distributed. Also, it is just two files

**Pull request merged**